airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/TestDestination.java

```diff
@@ -608,6 +610,51 @@ protected int getMaxRecordValueLimit() {
     return 1000000000;
   }
 
+  @Test
+  void testCustomDbtTransformations() throws Exception {
+    if (!implementsBasicNormalization()) {
```

Comment:
is this true only for now?

what if users want to run custom dbt transformations without normalisation? do we disallow this today?

Reply:
This is only for the tests to know if it should be ran or not. Nothing to do with configurations from users.

Both normalization and DBT operators depends on DBT to run.
Maybe if I renamed the function `implementsBasicNormalization` to `supportsDBT`it would make it easier to understand?